pkg/management/postgres/webserver/webserver.go

```diff
@@ -118,12 +118,26 @@ func (ws *Webserver) Start(ctx context.Context) error {
 	return nil
 }
 
+// sendTextResponse sends a generic text response
+//
+//nolint:unparam
```

Comment:
Linter complained that `statusCode` is always 200 and to remove the parameter. But it's not guaranteed to be so.